lib/nfg/capistrano/tasks/deploy.rake

```diff
@@ -0,0 +1,37 @@
+require 'aws-sdk-s3'
+
+namespace :deploy do
+  Rake::Task['deploy:compile_assets'].clear
+
+  desc 'Download or compile assets'
+  task :compile_assets => [:set_rails_env] do
+    assets_cache_prefix = Nfg::Capistrano::Config.circleci[:parameters][:assets_cache_prefix][:default]
+    revision = %x(git --no-pager log -1 --pretty=format:%H)
+    branch = fetch(:branch).gsub('/', '_')
+    assets_filename="#{assets_cache_prefix}-assets-#{revision}-#{branch}.tar.gz"
+    s3_bucket = Nfg::Capistrano::Config.circleci[:parameters][:s3_bucket][:default]
```

Comment:
I looked for the config.yml that had this but am having trouble finding it. Is it on a branch?

Reply:
Sorry, I should be clearer. I don't see a circleci/config.yaml file that has `:parameters :s3_bucket` parameter. Can you point to that?

Reply:
It is in this PR: https://github.com/network-for-good/donor_management/pull/3188/files

Reply:
great recalled that but couldn't find the pr, ty